parachain/pallets/ethereum-beacon-light-client/Cargo.toml

```
@@ -0,0 +1,64 @@
+[package]
+name = "snowbridge-ethereum-beacon-light-client"
+description = "Snowbridge Beacon Light Client Pallet"
+version = "0.0.1"
+edition = "2018"
+authors = ["Snowfork <contact@snowfork.com>"]
+repository = "https://github.com/Snowfork/snowbridge"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+serde = { version = "1.0.130", optional = true }
+codec = { version = "2.2.0", package = "parity-scale-codec", default-features = false, features = ["derive"] }
+scale-info = { version = "1.0", default-features = false, features = ["derive"] }
+hex = { package = "rustc-hex", version = "2.1.0", default-features = false }
+hex-literal = { version = "0.3.1", optional = true }
+rlp = { version = "0.5", default-features = false }
+eth2_ssz = { version = "0.4.0" }
+eth2_ssz_derive = { version = "0.3.0" }
+eth2_ssz_types = { version = "0.2.1" }
+tree_hash = { version = "0.4.0" }
+tree_hash_derive = { version = "0.4.0" }
+milagro_bls = { git = "https://github.com/sigp/milagro_bls", tag = "v1.4.2" }
```

This lib has an [std](https://github.com/sigp/milagro_bls/blob/16655aa033175a90c10ef02aa144e2835de23aec/Cargo.toml#L25) feature, which we should not enable by default.
```suggestion
milagro_bls = { git = "https://github.com/sigp/milagro_bls", tag = "v1.4.2", default-features = false }
```